Resources/public/js/views/services/ez-sectionserversideviewservice.js

```diff
@@ -143,5 +145,18 @@ YUI.add('ez-sectionserversideviewservice', function (Y) {
             });
             return action + '-' + sectionId + '-' + contentIds.join('_');
         },
+
+        /**
+         * Refreshes the section view
+         *
+         * @method _refreshView
+         * @protected
+         * @param {EventFacade} e
+         */
+        _refreshView: function (e) {
+            this.load(Y.bind(function () {
```

Comment:
you should probably set the app `loading` attribute to true so that the user has a visual feedback